packages/@aws-cdk/aws-backup/lib/rule.ts

```diff
@@ -156,5 +173,26 @@ export class BackupPlanRule {
     if (props.scheduleExpression && !/^cron/.test(props.scheduleExpression.expressionString)) {
       throw new Error('`scheduleExpression` must be of type `cron`');
     }
+
+    let deleteAfter = props.deleteAfter;
```

Comment:
How about `const deleteAfter = (props.enableContinuousBackup && !props.deleteAfter) ? Duration.days(35) : props.deleteAfter;` 

Just because I like `const` variables more than `let` whenever possible.